services/provider/consumerstatus/status.go

```diff
@@ -0,0 +1,41 @@
+package consumerstatus
+
+import (
+	pb "github.com/red-hat-storage/ocs-operator/v4/services/provider/pb"
+)
+
+type ConsumerStatus interface {
+	GetClusterVersion() string
+	GetOperatorVersion() string
+
+	SetClusterVersion(string) ConsumerStatus
+	SetOperatorVersion(string) ConsumerStatus
+}
+
+type StatusInfo struct {
+	*pb.ReportStatusRequest
```

Comment:
Further down the review, I can see we need to assign an existing Request structure to the wrapper so maybe keeping this as a pointer is not a bad approach (to prevent a copy operation)

Reply:
removed the wrapper struct